test/graphql.test.ts

```diff
@@ -1,3 +1,17 @@
+// Copyright © 2020 Atomist, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
why is this using the line comments? I thought that was changed to block comments.

Reply:
Bug, fixing currently. Do not merge until I fix it please.

Reply:
Well, fixing after standup